src/FixedDataTable.js

```diff
@@ -639,7 +640,8 @@ var FixedDataTable = React.createClass({
   },
 
   _renderRows(/*number*/ offsetTop) /*object*/ {
-    var state = this.state;
+    let state = updateRowHeights(this.state, this.props);
+    state = updateVisibleRows(state);
```

Comment:
Ya I wonder if we need to provide a hook for the user to invalidate row heights when something changes...  And if so do we want to invalidate all of them or just a subset?